docs/using_modin.rst

```diff
@@ -0,0 +1,74 @@
+Using Modin
+===========
+
+Modin is an early stage DataFrame library that wraps pandas and transparently
+distributes the data and computation, accelerating your pandas workflows with one line
+of code change. The user does not need to know how many cores their system has, nor do
+they need to specify how to distribute the data. In fact, users can continue using their
+previous pandas notebooks while experiencing a considerable speedup from Modin, even on
+a single machine. Only a modification of the import statement is needed, as we
+demonstrate below. Once you’ve changed your import statement, you’re ready to use Modin
+just like you would pandas, since the API is identical to pandas.
+
+.. code-block:: python
+
+  # import pandas as pd
+  import modin.pandas as pd
+
+Currently, we have part of the pandas API implemented and are working toward full
+functional parity with pandas.
+
+Using Modin on a Single Node
+----------------------------
+
+In order to use the most up-to-date version of Modin, please follow the instructions on
+the `installation page`_
+
+Once you import the library, you should see something similar to the following output:
+
+.. code-block:: text
+
+  >>> import modin.pandas as pd
+
+  Waiting for redis server at 127.0.0.1:14618 to respond...
+  Waiting for redis server at 127.0.0.1:31410 to respond...
+  Starting local scheduler with the following resources: {'CPU': 4, 'GPU': 0}.
+
+  ======================================================================
+  View the web UI at http://localhost:8889/notebooks/ray_ui36796.ipynb?token=ac25867d62c4ae87941bc5a0ecd5f517dbf80bd8e9b04218
+  ======================================================================
+
+Once you have executed  ``import modin.pandas as pd``, you're ready to begin
+running your pandas pipeline as you were before.
+
+APIs Supported
+--------------
+
+Please note, the API is not yet complete. For some methods, you may see the following:
+
+.. code-block:: text
+
+  NotImplementedError: To contribute to Pandas on Ray, please visit github.com/modin-project/modin.
```

Comment:
Should we change the message for `NotImplementedError`'s to say Modin instead of Pandas on Ray?

Reply:
Probably, do you want to submit the PR for that?

Reply:
Sure, I'll submit a PR